controllers/datadogagent/common_configmap.go

```diff
@@ -68,6 +68,7 @@ func (r *Reconciler) updateIfNeededConfigMap(dda *datadoghqv1alpha1.DatadogAgent
 	// Copy possibly changed fields
 	updateCM := oldConfigMap.DeepCopy()
 	updateCM.Data = newConfigMap.Data
+	// TODO if new CM does not have labels it panics.
```

Comment:
what is this todo?

Reply:
I know :( I will add a card to investigate.